README.md

```diff
@@ -41,6 +41,9 @@ As an alternative to the section above, you can run the app within a Docker cont
 
 * `$ docker run -p 5000:5000 -e "GITHUB_TOKEN=YOUR_TOKEN" hacktoberfest-checker`
 
+## Link
+Check your progress on [Hacktoberfest official site](https://hacktoberfest.digitalocean.com/stats/)
```

Comment:
I'm not sure about this, it might be confusing having a link to an alternative checker in the README. If we are going to add it, we should explain better what it is and how it is different. What do you think?